src/base/http/requestparser.cpp

```diff
@@ -308,16 +308,16 @@ bool RequestParser::parsePostMessage(const QByteArray &data)
 
 bool RequestParser::parseFormData(const QByteArray &data)
 {
-    const QVector<QByteArray> list = splitToViews(data, EOH, Qt::KeepEmptyParts);
+    const int eohPos = data.indexOf(EOH);
 
-    if (list.size() != 2)
+    if (eohPos < 0)
     {
         qWarning() << Q_FUNC_INFO << "multipart/form-data format error";
         return false;
     }
 
-    const QString headers = QString::fromLatin1(list[0]);
-    const QByteArray payload = viewWithoutEndingWith(list[1], CRLF);
+    const QString headers = data.left(eohPos);
```

Comment:
Please use `Utils::ByteArray::midView` to avoid a copy, and in the below line too.

Reply:
Switched to `Utils::ByteArray::midView` as suggested.